provider/provider.go

```diff
@@ -580,3 +582,30 @@ func getIgnore(dockerIgnorePath string) ([]string, error) {
 	}
 	return ignorePatterns, nil
 }
+
+// setConfiguration takes in the stack config settings and additionally reads in any environment variables
+func setConfiguration(configVars map[string]string) map[string]string {
```

Comment:
You can link here https://github.com/pulumi/pulumi-terraform-bridge/issues/1238 with an explanation that if that was built, parsing env vars again here would not be required.